views/partials/breach-detail-content-group.hbs

```diff
@@ -1,14 +1,29 @@
-              <h2 class="section-headline txt-purple7">{{ this.headline }}</h2>
-
+              
+            <div class="mw-700 breach-detail {{ sectionId }}">
+              {{#if smallerSectionHeadline}}
+                <h3 class="small-section-headline txt-purple7 bold">{{ this.headline }}</h3>
+              {{else}}
+                <h2 class="section-headline txt-purple7">{{ this.headline }}</h2>
+              {{/if}}
```

Comment:
This feels wrong. I don't think you're supposed to jump headings for a11y reasons.
Not sure if we should always just use an `<h2>` here and just change the classes to make the UI different, vs toggling between an H2 or H3 (unless we know they'll always be in a logical H1=>H2=>H3 order, in which case feel free to ignore me).